packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1426,25 +1430,26 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
   }
 }
 
-/// A convenience widget that wraps a [DropdownButton] in a [FormField].
+/// A convenience widget that makes a [DropdownButton] into a [FormField].
 class DropdownButtonFormField<T> extends FormField<T> {
-  /// Creates a [DropdownButton] widget wrapped in an [InputDecorator] and
+  /// Creates a [DropdownButton] widget wrapped in an [InputDecorator] that is a
```

Comment:
This is a little hard to parse, it sounds like the InputDecorator is a FormField.

Reply:
OK, reworded it.